src/chrome/content/rules/University-of-Exeter.xml

```diff
@@ -26,6 +30,10 @@
 	<securecookie host="^(?:.*\.)?exeter\.ac\.uk$" name=".+" />
 
 
+	<!-- Cert only matches empslocal.ex.ac.uk -->
+	<rule from="^http://empslocal\.ex\.ac\.uk/"
+		to="https://empslocal.ex.ac.uk/" />
+
 	<rule from="^http://newton\.ex\.ac\.uk/"
 		to="https://newton.ex.ac.uk/" />
```

Comment:
Looks like this one should rewrite to newton.exeter.ac.ukas the cert is only for the full version...

```
This server could not prove that it is emps.ex.ac.uk; its security certificate is from *.exeter.ac.uk. This may be caused by a misconfiguration or an attacker intercepting your connection.
```